tasks.mak

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Google LLC
```

Comment:
Can we add some documentation of this tool to:
* https://github.com/GoogleCloudPlatform/ops-agent/tree/master/dev-docs (e.g. a new `dev-tool.md` file)
* Internal dev guide (can point to the new `.md` file above)